plugins/modules/net_tools/nmcli.py

```diff
@@ -279,6 +279,20 @@
             - When updating this property on a currently activated connection, the change takes effect immediately.
        type: str
        version_added: 2.0.0
+    wifi_sec:
+       description:
+            - 'The security configuration of the Wifi connection. The valid attributes are listed on:'
+            - 'https://developer.gnome.org/NetworkManager/stable/settings-802-11-wireless-security.html'
+            - 'For instance to use common WPA-PSK auth with a password:'
+            - '  - key-mgmt: wpa-psk'
+            - '  - psk: my_password'
```

Comment:
This should be a dictionary, not a list of dictionaries, right? It's probably better to format it like this:
```suggestion
            - 'C({key-mgmt: wpa-psk, psk: my_password})'
```